maven/fetch.bzl

```diff
@@ -38,14 +38,7 @@ def _format_exported_files(paths):
 def _get_packaging_type(ctx, pom_label):
     packaging_type_file = pom_label.relative(":%s" % PACKAGING_TYPE_FILE)
     path = ctx.path(packaging_type_file)
-    result = ctx.execute(["cat", path])  # TODO(cgruber) Convert to read post Bazel 0.29
-    if result.return_code != 0:
-        fail("Failed to retreive packaging type from %s for artifact %s: %s" % (
-            packaging_type_file,
-            ctx.attr.artifact,
-            result.stderr,
-        ))
-    return strings.trim(result.stdout)
+    return strings.trim(ctx.read(path))
```

Comment:
crazy that to do a read you had to do a cat

Reply:
Yeah. Originally the read() API didn't exist.  Then it did, but only could work within the boundaries of the current repository rule's workspace. Now it's broader, so I don't have to work around it.